war/src/main/js/config-tabbar.js

```diff
@@ -94,3 +98,22 @@ function addFinderToggle(configTableMetadata) {
 function fireBottomStickerAdjustEvent() {
     Event.fire(window, 'jenkins:bottom-sticker-adjust'); // jshint ignore:line
 }
+// YUI Drag widget does not like to work on elements with a relative position.
+// This tells the element to switch to static position at the start of the drag, so it can work.
+function fixDragEvent(handle){
+    var isReady = false;
+    var $handle = $(handle);
+    var $chunk = $handle.closest('.repeated-chunk');
+    $handle.add('#ygddfdiv')
+    	.mousedown(function(){
+    	    isReady = true; 
+    	})
+    	.mousemove(function(){
+    	    if(isReady && !$chunk.hasClass('dragging')){
+    		$chunk.addClass('dragging');
+    	    }
+    	}).mouseup(function(){
+    	    isReady = false;
+    	    $chunk.removeClass('dragging');
+    	});
+}
```

Comment:
No need to do it now, but it might make sense to eventually put this function into a module of its own.